2025-05-25 21:32:16,688 INFO    MainThread:136312 [wandb_setup.py:_flush():70] Current SDK version is 0.19.11
2025-05-25 21:32:16,688 INFO    MainThread:136312 [wandb_setup.py:_flush():70] Configure stats pid to 136312
2025-05-25 21:32:16,688 INFO    MainThread:136312 [wandb_setup.py:_flush():70] Loading settings from /home/david-lacour/.config/wandb/settings
2025-05-25 21:32:16,688 INFO    MainThread:136312 [wandb_setup.py:_flush():70] Loading settings from /home/david-lacour/Documents/transformerVision/githubs/VizInt/autoflex/autotinymodels/wandb/settings
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_setup.py:_flush():70] Loading settings from environment variables
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /home/david-lacour/Documents/transformerVision/githubs/VizInt/autoflex/autotinymodels/wandb/run-20250525_213216-gzjyhhad/logs/debug.log
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /home/david-lacour/Documents/transformerVision/githubs/VizInt/autoflex/autotinymodels/wandb/run-20250525_213216-gzjyhhad/logs/debug-internal.log
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_init.py:init():852] calling init triggers
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_init.py:init():893] starting backend
2025-05-25 21:32:16,689 INFO    MainThread:136312 [wandb_init.py:init():897] sending inform_init request
2025-05-25 21:32:16,693 INFO    MainThread:136312 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-25 21:32:16,693 INFO    MainThread:136312 [wandb_init.py:init():907] backend started and connected
2025-05-25 21:32:16,695 INFO    MainThread:136312 [wandb_init.py:init():1005] updated telemetry
2025-05-25 21:32:16,702 INFO    MainThread:136312 [wandb_init.py:init():1029] communicating run to backend with 90.0 second timeout
2025-05-25 21:32:16,883 WARNING MainThread:136312 [wandb_init.py:init():1681] [no run ID] interrupted
Traceback (most recent call last):
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1677, in init
    return wi.init(run_settings, run_config, run_printer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1055, in init
    result = wait_with_progress(
             ^^^^^^^^^^^^^^^^^^^
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 70, in wait_all_with_progress
    return _wait_handles(handle_list, timeout=progress_after)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 110, in _wait_handles
    results.append(handle.wait_or(timeout=remaining_timeout))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/mailbox/mailbox_handle.py", line 122, in wait_or
    return self._fn(self._handle.wait_or(timeout=timeout))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/david-lacour/.pyenv/versions/myenv/lib/python3.12/site-packages/wandb/sdk/mailbox/response_handle.py", line 88, in wait_or
    if not self._event.wait(timeout=timeout):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2025-05-25 21:32:17,456 INFO    MsgRouterThr:136312 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
